recipes/qt/6.x.x/conanfile.py

```diff
@@ -624,7 +621,12 @@ def generate(self):
         if self.options.cross_compile:
             tc.variables["QT_QMAKE_DEVICE_OPTIONS"] = f"CROSS_COMPILE={self.options.cross_compile}"
         if cross_building(self):
-            tc.variables["QT_HOST_PATH"] = self.dependencies.direct_build["qt"].package_folder
+            # Mainly to locate Qt6HostInfoConfig.cmake
+            tc.cache_variables["QT_HOST_PATH"] = self.dependencies.direct_build["qt"].package_folder
+            # Stand-in for Qt6CoreTools - which is loaded for the executable targets
+            tc.cache_variables["CMAKE_PROJECT_Qt_INCLUDE"] = os.path.join(self.dependencies.direct_build["qt"].package_folder, self._cmake_executables_file)
+            # Ensure tools for host are always built
```

Comment:
What's the reason for this? It makes the build not work on iOS since the tools doesn't build for that platform.